Testing/xASL_ut_UnitTesting.m

```diff
@@ -57,20 +57,23 @@
 
     %% Test Workflow
 
+    % Unit test: xASL_test_GetLogContent
+    UnitTests(1) = xASL_ut_UnitTest_function_GetLogContent(TestRepository);
+
     % Unit test: xASL_tsvRead
-    UnitTests(1) = xASL_ut_UnitTest_function_tsvRead(TestRepository);
+    UnitTests(size(UnitTests,2)+1) = xASL_ut_UnitTest_function_tsvRead(TestRepository);
```

Comment:
The problem was that I didn't want to use feval or eval, since this would make testing harder for no real gained benefit. Wrote a fix with function handles now, if you prefer that. Check out 17b1103.